iot_config/src/route.rs

```diff
@@ -523,18 +523,22 @@ pub fn list_devaddr_ranges_for_route<'a>(
 
 pub fn active_route_stream<'a>(
     db: impl sqlx::PgExecutor<'a> + 'a,
-) -> impl Stream<Item = Route> + 'a {
-    sqlx::query_as::<_, StorageRoute>(
+    since: &'a DateTime<Utc>,
```

Comment:
do you need the lifetime/reference on the datetime? DateTime implements Copy since it's probably just an integer under the hood

Reply:
Addressed in 1390ffc